package.json

```diff
@@ -15,8 +15,8 @@
     "test": "npm run test:unit && npm run test:functional",
     "test:unit": "karma start --single-run",
     "test:unit:watch": "karma start",
-    "test:unit:browserstack:local": "karma start karma.browserstack.conf.js",
-    "test:unit:browserstack:ci": "karma start karma.browserstack.conf.js --browsers bs_chrome_windows,bs_safari_macos,bs_firefox_windows,bs_ie_windows --single-run --reporters spec,BrowserStack",
```

Comment:
The arguments we had here were unnecessary, since they were already set in karma.browserstack.conf.js. You may notice that both `test:unit:browserstack:local` and `test:unit:browserstack:ci` are the exact same now. We could consolidate them, but I think I prefer keeping them separate just to be clear that CI depends on it and `test:unit:browserstack:local` can potentially be tweaked independently.